lua/lazyvim/plugins/extras/lang/angular.lua

```diff
@@ -31,7 +31,12 @@ return {
     "neovim/nvim-lspconfig",
     opts = {
       servers = {
-        angularls = {},
+        angularls = {
+          -- Make sure it works for Nx workspaces as well
+          root_dir = function(fname)
+            return require("lspconfig.util").root_pattern("nx.json", "angular.json")(fname) or vim.loop.cwd()
```

Comment:
Why is `vim.loop.cwd` needed here? Shouldn't the `root_dir` just be resolved based on the patterns? What will happen if it doesn't find a pattern? Will the LSP server still attach based on the `cwd`?

Reply:
removed it, without it also works the root folder detection

Reply:
I don't do any webdev and like I've told you before, I'm just a simple user like you. So, this was more of a suggestion for you to test it out, since I can't test it.

About the review, I don't feel confident enough to officially approve it but it looks good to me. 

The maintainer will do the final review and will be the one to approve it. Good luck :smile: